platform/viewer/src/connectedComponents/ConnectedDownloadDialog.js

```diff
@@ -0,0 +1,43 @@
+import { connect } from 'react-redux';
```

Comment:
This shouldn't need to be a connected component. All of it's state should be local.

Reply:
Please, could you send me an example of how to connect to the ToolbarRow without connect it? 

I followed what you advice with Cine  Dialog, if there is a better way to do this, would be nice to update cine dialog connection too.